.gitignore

```diff
@@ -37,3 +37,6 @@ public/assets/*.json
 
 # don't ignore these css
 !public/assets/error-pages.85f433bf6332a1245653b92b44d2430a55810be0.min.css
+
+# Dotenv
+.env
```

Comment:
I don't think we'll need to use a `.env` file, or use the `dotenv-rails` gem.  GOV.UK PaaS lets us define environment variables (and therefore secrets) for each application: https://docs.cloud.service.gov.uk/deploying_apps.html#environment-variables.  We should be able to get a key for staging and another for production, then add them using the CloudFoundry CLI.

Reply:
Thanks @brucebolt - Yes, quite right - `dotenv` isn't required, so has been removed.